tox.ini

```
@@ -39,6 +39,8 @@ basepython = {[testenv:py3]basepython}
 
 [testenv:py3-functional]
 basepython = {[testenv:py3]basepython}
+passenv =
+    TEST_DIR
```

I think this is the right syntax to pass the `passenv` settings from the parent section:
```suggestion
passenv =
    {[testenv]passenv}
    TEST_DIR
```

---

It's working for me without this.

---

Can you pass a modified CMAKE_ARGS without it?

---

Why would I do that? This is just for running the func test. Am I missing something?

---

Added.

---

I think otherwise there's no way to run functional tests via `tox` with CUDA etc.